zipkin-server/src/main/java/zipkin2/server/internal/throttle/ThrottledCall.java

```diff
@@ -38,11 +38,21 @@
  * @see ThrottledStorageComponent
  */
 final class ThrottledCall extends Call.Base<Void> {
```

Comment:
An interesting, GC related, aspect of this change is that you technically may have increased the memory footprint of this class while keeping the EnqueueAndAwait class the same (removed a member variable but made it non-static).  Technically speaking, there is a chance that would make GC worse but the benchmarks are reasonably clear at the moment.

Either way, one thing that would be interesting to see is if you ran [JoL](https://openjdk.java.net/projects/code-tools/jol/) against the two instances (look at the samples, it's pretty easy to use) and see if any concerning differences pop up.